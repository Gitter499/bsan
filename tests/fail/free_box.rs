//@run: 1
fn main() {
    let b = Box::new(1);

    let ptr = Box::into_raw(b);

    unsafe {
        drop(Box::from_raw(ptr));
    }

    unsafe {
        *ptr = 1;
    }
}
