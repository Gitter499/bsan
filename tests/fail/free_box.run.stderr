An error occurred: UndefinedBehavior(UseAfterFree(ALLOC))


